src/open_inwoner/openzaak/models.py

```diff
@@ -300,6 +302,63 @@ def __str__(self):
         return self.omschrijving
 
 
+class ZaakTypeStatusTypeConfig(models.Model):
+    zaaktype_config = models.ForeignKey(
+        "openzaak.ZaakTypeConfig",
+        on_delete=models.CASCADE,
+    )
+    statustype_url = models.URLField(
+        verbose_name=_("Statustype URL"),
+        max_length=1000,
+    )
+    omschrijving = models.CharField(
+        verbose_name=_("Omschrijving"),
+        max_length=80,
+    )
+    statustekst = models.CharField(
+        verbose_name=_("Statustekst"),
+        max_length=1000,
+    )
+    zaaktype_uuids = ArrayField(
+        models.UUIDField(
+            verbose_name=_("Zaaktype UUID"),
+        ),
+        default=list,
+    )
+
+    # configuration
+    status_indicator = models.CharField(
+        blank=True,
+        max_length=32,
+        choices=StatusIndicators.choices,
+        verbose_name=_("Statustype indicator"),
+        help_text=_(
+            "Determines what will be shown to the user if a case is set to this status"
+        ),
+    )
+    status_indicator_text = models.TextField(
```

Comment:
Not sure how we deal with translations at the moment

Reply:
We don't, OIP is Dutch only